app/i18n/messages/en.json

```diff
@@ -345,6 +345,7 @@
   "ResourceAvailability.availableTime": "Available until {closesAt}",
   "ResourceAvailability.closed": "Closed on selected day",
   "ResourceAvailability.closedToday": "Closed today",
+  "ResourceAvailability.externalReservation": "Reservations otherwise --english--",
```

Comment:
"Reservations externally"

Not 100% sure about this either without context for the string.